managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx

```diff
@@ -0,0 +1,464 @@
+
+    ```
+    SELECT * from tap-postgres.{table1name} LIMIT 10;
+    ```
+    </Message>
+
+## Importing data from an existing PostgreSQL backup
+
+### Using pg_restore
+
+#### Prerequisites
+
+To complete the following procedure, you must have:
+
+- Installed PostgreSQL 16 (or newer) with [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) (bundled with the default PostgreSQL installation).
+
+- A backup file for your database (named `my-backup` in the following procedure). 
+
+1. In a terminal, access the directory containing the backup file, then run the command below to import data to your Managed Database for PostgreSQL using `pg_restore`. Make sure to replace the placeholders with your Managed Database for PostgreSQL connection parameters:
+
+    ```
+    pg_restore --no-privileges --no-owner --clean --if-exists -U {username} -h {host} --port 5432 -d {databasename} my-backup
+    ```
+
+    <Message type="note">
+    You can find this information on the [overview page](https://console.scaleway.com/rdb/instances) of your database. The password is the [IAM secret key](/identity-and-access-management/iam/concepts/#api-key) corresponding to the [IAM user](/identity-and-access-management/iam/concepts/#user) or [IAM application](/identity-and-access-management/iam/concepts/#application) you are connecting with. 
+    </Message>
+
+2. When finished, make sure your data is stored in your new database by [connecting to it](/managed-databases/postgresql-and-mysql/how-to/connect-database-instance/), and performing a query.
+
+## Import Data from a file (.csv, .txt, etc.)
+
+### Using psql
+
+#### Prerequisites
+
+To complete the following procedure, you must have:
+
+- Installed PostgreSQL 16 (or newer) with [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) (bundled with the default PostgreSQL installation).
+
+- A data file corresponding to a single table (named named `my-table.csv` in the following procedure).
+
+<Message type="tip">
+You can create a `.csv` file from an existing PostgreSQL table with the [COPY TO command](https://www.postgresql.org/docs/current/sql-copy.html), as shown below:
+
+```sql
+\COPY {tablename} TO './my-table.csv' WITH DELIMITER ',' CSV;
+```
+</Message>
+
+1. In a terminal, access the folder containing your data file:
+
+    ```
+    cd path/to/my-table.csv
+    ```
+
+2. Connect to your Managed Database for PostgreSQL using `psql`:
+
+    ```sh
+    psql "postgresql://{username}:{password}@{host}:5432/{databasename}?sslmode=require"
+    ```
+
+    <Message type="note">
+    You can find this information on the [overview page](https://console.scaleway.com/rdb/instances) of your database. The password is the [IAM secret key](/identity-and-access-management/iam/concepts/#api-key) corresponding to the [IAM user](/identity-and-access-management/iam/concepts/#user) or [IAM application](/identity-and-access-management/iam/concepts/#application) you are connecting with. 
```

Comment:
Managed PostgreSQL is not integrated with IAM yet so we need to replace this with "the username & password that you defined upon Database Instance Creation"